platform/ios/app/MBXEmbeddedMapViewController.m

```diff
@@ -0,0 +1,188 @@
+#import "MBXEmbeddedMapViewController.h"
+#import <Mapbox/Mapbox.h>
+
+typedef NS_ENUM(NSInteger, MBXEmbeddedControl) {
+    MBXEmbeddedControlZoom = 0,
+    MBXEmbeddedControlScroll,
+    MBXEmbeddedControlRotation,
+    MBXEmbeddedControlPitch
+};
+
+
+@interface MBXEmbeddedMapViewController () <UIScrollViewDelegate>
+
+@property (nonatomic, strong) MGLMapView *mapView;
+
+@end
+
+@implementation MBXEmbeddedMapViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    UIRotationGestureRecognizer *gestureRecognizer = [[UIRotationGestureRecognizer alloc] initWithTarget:self action:@selector(rotation:)];
+    
+    UIScrollView *scrollView = [[UIScrollView alloc] initWithFrame:self.view.frame];
+    scrollView.delegate = self;
+    [self.view addSubview:scrollView];
+    [scrollView addGestureRecognizer:gestureRecognizer];
+    
+    CGRect mapRect = CGRectMake(0, 0, CGRectGetWidth(scrollView.frame) * 2, CGRectGetHeight(scrollView.frame) * 2);
```

Comment:
I’m not following the purpose of a gigantic scrollview — is this a common use case that we’re demoing? It seems like multiple smaller maps within a regular viewport frame (with only a vertical scroll axis) is a potential use case that would demonstrate these conditionally disabled gestures in a more straightforward way.